src/main/java/ogc/rs/apiserver/ApiServerVerticle.java

```diff
@@ -2121,8 +2121,8 @@ public void updateStacCollection(RoutingContext routingContext) {
                     if (requestBody.containsKey("crs")) {
                         existingCollection.put("crs", requestBody.getString("crs"));
                     }
-                    if (requestBody.containsKey("datetimeKey")) {
-                        existingCollection.put("datetimeKey", requestBody.getString("datetimeKey"));
+                    if (requestBody.containsKey("datetime")) {
```

Comment:
We don't need this. The `requestBody` will not contain any information regarding `datetimeKey`